app/models/trainee.rb

```diff
@@ -424,6 +424,12 @@ def available_courses(training_route = self.training_route)
     provider.courses.where(route: training_route).order(:name) if TRAINING_ROUTES_FOR_COURSE.keys.include?(training_route)
   end
 
+  def current_course
```

Comment:
let's remove this method now that we aren't using it to avoid future confusion about what it does